doc/urdf_srdf/urdf_srdf_tutorial.rst

```diff
@@ -9,7 +9,7 @@ URDF Resources
 ^^^^^^^^^^^^^^
 
 * `URDF ROS Wiki Page <http://www.ros.org/wiki/urdf>`_ - The URDF ROS Wiki page is the source of most information about the URDF.
-* `URDF Tutorials <http://www.ros.org/wiki/urdf/Tutorials>`_ - Tutorials for working with the URDF.
+* `URDF Tutorials <https://docs.ros.org/en/DISTRO/Tutorials/URDF/URDF-Main.html>`_ - Tutorials for working with the URDF.
```

Comment:
This is incorrect, but I wasn't sure how to properly link to https://docs.ros.org/en/rolling/Tutorials/URDF/URDF-Main.html

Reply:
for now I think it is better to just link the rolling documentation and long term we'll figure out a solution to make this ros version specific.